cli/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [0.53.3] - 2024-01-10
```

Comment:
I think it's 0.52.3?

Reply:
good spot - too many numbers floating about apparently